src/components/Form/PinInput/PinInput.tsx

```diff
@@ -85,18 +98,36 @@ export const PinInput = forwardRef<HTMLElement, PinInputProps>(({
         </header>
         <div className={styles.buttonWrapper}>
           {createChunks(AVAILABLE_PINS, 3).map((rows) => (
-            <div key={rows.toString()} className={styles.row}>
+            <div key={rows.toString()} className={classNames(
+              styles.row,
+              biometricType === undefined && styles['row--no-biometric'],
+            )}>
               {rows.map((element) => {
                 let children: ReactNode = element;
-                let clickFunction = () => handleClickValue(Number(element));
+                let clickFunction = () => {
+                  if (typeof element === 'number') {
+                    handleClickValue(element);
+                  }
+                };
 
                 if (element === Keys.BACKSPACE) {
                   clickFunction = () => handleClickBackspace();
                   children = <Icon36Backspace className={styles.backspaceIcon} />;
+                } else if (element === BIOMETRIC_AUTH) {
```

Comment:
It's time to make this in a separate function. Code became too big for this scope. Something like `const [children, callbackFunction] = ...`

Reply:
There are no different properties for `biometricIcon`, so you can easily do something like `<Component className={styles.actionButton} />`, where `Component` is value from switch